__tests__/run.test.ts

```diff
@@ -0,0 +1,36 @@
+import { fromLiteralsToFromFile } from "../src/run"
+import { createFile } from '../src/file.utility';
+import { mocked } from 'ts-jest/utils'
+
+jest.mock('../src/file.utility');
+
+const fileUtility = mocked(createFile, true)
+
+test('Literal converted to file', () => {
+    fileUtility.mockReturnValue("./key");
+    expect(fromLiteralsToFromFile("--from-literal=key=value")).toBe(' --from-file=./key')
+})
+
+test('File maintained as-is', () => {
+    fileUtility.mockReturnValue("./key");
+    expect(fromLiteralsToFromFile("--from-file=./filepath")).toBe(' --from-file=./filepath')
+})
+
+test('Any other arguments maintained as-is', () => {
+    fileUtility.mockReturnValue("./key");
+    expect(fromLiteralsToFromFile("--otherArgument=value")).toBe(' --otherArgument=value')
+})
+
+test('Invalid case, no value for secret', () => {
+    expect(() => fromLiteralsToFromFile("--from-literal=key")).toThrow(Error);
+})
+
+test('Multiple commnads combined', () => {
+    fileUtility.mockReturnValue("./key");
+    expect(fromLiteralsToFromFile("--from-literal=key=value --from-file=./filepath --otherArgument=value"))
+        .toBe(' --from-file=./key --from-file=./filepath  --otherArgument=value')
```

Comment:
Is this assert working? This seems to have an extra space/

Reply:
Yep. Added a trim to remove any extra space.